src/www/ui/api/documentation/openapi.yaml

```
@@ -1302,6 +1302,40 @@ paths:
 
         default:
           $ref: '#/components/responses/defaultResponse'
+
+  /file/upload/{id}/item/{ItemId}/copyrights:
```

Please be consistent with other endpoints.
```suggestion
  /uploads/{id}/item/{itemId}/copyrights:
```